.github/workflows/release.yml

```diff
@@ -27,6 +27,9 @@ jobs:
         if: steps.branch.outputs.BRANCH_NAME == 'main'
         run: |
           make build-all-small
+      - name: Create CHECKSUMS
+        if: steps.branch.outputs.BRANCH_NAME == 'main'
+        run: cd cli/dist ; sha256sum -b carrier* > SHA256SUM.txt
```

Comment:
This is missing the `( )` of my original proposal. As is this `cd` affects the steps after. With the parens it will be a sub shell limiting the scope of the `cd` to that command and step.

Reply:
Danke. I didn't know that info.